guidelines/sc/22/focus-appearance-minimum.html

```diff
@@ -5,11 +5,11 @@ <h4>Focus Appearance (Minimum)</h4>
     <p class="conformance-level">AA</p>
 	<p class="change">New</p>
     
-    <p>For the keyboard focus indicator of each User Interface Component, all of the following are true:</p>
+    <p>When user interface components receive keyboard focus, all of the following are true:</p>
     <ul>
-        <li><strong>Minimum area:</strong> The <a>focus indication area</a> is greater than or equal to a 1 <a>CSS pixel</a> border of the focused control, or has a thickness of at least 8 CSS pixels along the shortest side of the element.</li>
-        <li><strong>Change of contrast:</strong> The color change for the focus indication area has a contrast ratio of at least 3:1 with the colors of the unfocused state.</li>
-        <li><strong>Adjacent contrast:</strong> The focus indication area has a contrast ratio of at least 3:1 against all adjacent colors for the minimum area or greater, or has a thickness of at least 2 CSS pixels.</li>
+        <li><strong>Minimum area:</strong> The area of the focus indicator is at least as large as the area of a 1 <a>CSS pixel</a> thick perimeter of the focused control, or the focus indicator has a thickness of at least 8 CSS pixels along the shortest side of the user interface component.</li>
+        <li><strong>Change of contrast:</strong> The pixels making up the minimum focus indicator area achieve at least a 3:1 contrast between their colors in the focused and unfocused states. </li>
+        <li><strong>Adjacent contrast:</strong> The focus indicator contrasts at least 3:1 against all adjacent colors, or has a thickness of at least 2 CSS pixels.</li>
         <li><strong>Not fully obscured:</strong> The item with focus is not entirely hidden by author-created content.</li>
```

Comment:
I think most/all of these changes have already gone in, we'll just have to check for merge issues.